argo-events/overlays/dh-prod-thanos-extractor/kustomization.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: dh-prod-thanos-extractor
```

Comment:
@HumairAK we don't need to specify namespace here do we?

Reply:
fwiw it is specified in every other overlays' `kustomization.yaml` files (at least for argo)

Reply:
@anishasthana looks fine to me